pkg/opensearch/client/search_request.go

```diff
@@ -142,6 +142,31 @@ func NewMultiSearchRequestBuilder(flavor Flavor, version *semver.Version) *Multi
 	}
 }
 
+func (b *SearchRequestBuilder) SetTraceListFilters(to, from int64, query string) {
```

Comment:
This sets up the "query" object of the query to OpenSearch. See the snapshot [here](https://github.com/grafana/opensearch-datasource/pull/284/files#diff-a0ae264a941e1be77e2c5deab9b0e33792443dee84d614daa9b009fe4717e786R42).